packages/@okta/vuepress-theme-default/global-components/Breadcrumb.vue

```diff
@@ -26,6 +26,14 @@
           return crumbs;
         }
 
+        if(this.$page.path.startsWith('/docs/reference/')) {
+          crumbs.push(
+            {path: '/docs/', title: 'Docs'},
+            {path: '/docs/reference/', title: 'Reference' },
+          );
+          return crumbs;
+        }
+
         if(this.$page.path == '/reference/' || this.$page.path.startsWith('/code/')) {
```

Comment:
Does this `/reference/` check remain valid?

Reply:
technically, yes.  we will remove this when we migrate fully to the new design